### ISSUE TITLE
Integrate the allure report into Rspec tests with Jenkins

### ISSUE THREAD

**Issue by vnicolev** [allure-framework/allure-rspec#77]
Hi folks,

I am trying to integrate the allure report into Rspec tests with Jenkins, but the Jenkins Console output keep saying:

```
16:37:10 Exception in thread "main" ru.yandex.qatools.allure.data.ReportGenerationException: Could not find any allure results
16:37:10 	at ru.yandex.qatools.allure.data.AllureReportGenerator.generate(AllureReportGenerator.java:58)
16:37:10 	at ru.yandex.qatools.allure.data.AllureReportGenerator.generate(AllureReportGenerator.java:53)
16:37:10 	at ru.yandex.qatools.allure.AllureMain.main(AllureMain.java:48)
16:37:10 Command aborted due to exception {}.
16:37:10 org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)
16:37:10 	at org.apache.commons.exec.DefaultExecutor.executeInternal(DefaultExecutor.java:404)
16:37:10 	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:166)
16:37:10 	at org.apache.commons.exec.DefaultExecutor.execute(DefaultExecutor.java:153)
16:37:10 	at ru.yandex.qatools.allure.command.ReportGenerate.runUnsafe(ReportGenerate.java:48)
16:37:10 	at ru.yandex.qatools.allure.command.AbstractCommand.run(AbstractCommand.java:52)
16:37:10 	at ru.yandex.qatools.allure.CommandLine.main(CommandLine.java:46)
16:37:11 Starting web server for report directory <allure-report>
16:37:11 2017-06-21 16:37:11.343:INFO::main: Logging initialized @843ms
16:37:11 Can not open browser because this capability is not supported on your platform. You can use the link below to open the report manually.
16:37:11 Server started at <http://172.31.14.172:42798/>. Press <Ctrl+C> to exit ...
```

Here's the part of the command in the execute shell:
```  
  bundle install
  bundle exec rake db:migrate
  export PATH=~/.yarn/bin:$PATH
  cd client && yarn install && npm run build && cd ..
  yarn install
  RAILS_ENV=test bin/webpack
  bundle exec rspec  
  allure generate allure-results/ -o allure-report/
  allure report open -o allure-report/
```

In our Ubuntu server, I can see 'allure-results' and 'allure-report' under the root of the project, but no xml files in them. 
Under 'allure-results' folder, only 'executor.json' and 'testrun.json' exist; 
and under 'allure-report', only 'data' and 'plugins' exist. 

However, it runs perfectly on my local Mac 10.12. I'm using rspec 3.5.

Does anyone know how to solve this problem?

**Comment by MacCracken** [comment]
Have you tried using the Allure-Report Plugin for Jenkins, it does the last two steps as a post-build execution.

**Comment by vnicolev** [comment]
Thanks for the suggestion!
Yes, I've installed Allure-Report Plugin, and after I removed the last two steps, it worked!

But the layout looks a little bit strange.

The local report:
![2017-06-22 3 48 23](https://user-images.githubusercontent.com/26741862/27422981-aed1d6d8-5762-11e7-98b2-46006e086fae.png)

The generated report on Jenkins:
![2017-06-22 3 48 54](https://user-images.githubusercontent.com/26741862/27423022-d3f6a6d2-5762-11e7-8e20-071e48b28c49.png)

I have no idea why there's an unexpected token error. I am new to the allure report. Hope it's not a stupid question.
